### DISCUSSION TITLE
Script Internet Access?

### DISCUSSION THREAD

**Discussion by antgiant** [Stellarium/stellarium#1872] [Q&A] [answered]
I am writing a script that ideally would utilize data from a web URL for some of its functionality. Is it possible for scripts to make AJAX calls (or even synchronous calls￼) to pull in data? So far I haven't been able to figure out how.

**Comment by alex-w** [comment, marked as answer]
Sorry, it’s not possible, because this is not implemented by some reasons.

**Reply by antgiant** [reply]
That's a bummer. Thanks.

Is it possible to safely load data from a file then? Then best I've been able to figure out so far is include('file name');. However, that requires the file be entirely valid JavaScript as it actively breaks try catch somehow, and takes out the entire script if there is any error.
